desktop/reportApp/main.cpp

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2017-present, Status Research and Development GmbH.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <QDebug>
+#include <QGuiApplication>
+#include <QQmlContext>
+#include <QQuickView>
+
+#include "reportpublisher.h"
+
+const int MAIN_WINDOW_WIDTH = 1024;
+const int MAIN_WINDOW_HEIGHT = 768;
+const int INPUT_ARGUMENTS_COUNT = 5;
+
+int main(int argc, char **argv) {
+
+  QGuiApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
+  QGuiApplication app(argc, argv);
+
+  qDebug() << argc << " args[1]: " << argv[1] << " args[2]: " << argv[2];
```

Comment:
Do we need this?

Reply:
@vkjr not really, will remove this, thanks